add new usage limits page (#786)
humanmade/altis-cloud: docs/usage.md

> # Usage Limits

Reviewer: Let's rename this to "Usage"
```suggestion
# Usage
```

Each Altis environment is tailored to fit your usage, giving your customers a great experience while ensuring you only pay for what you need. Altis measures your traffic usage based on pages served to your users.

Altis environments are designed to scale to meet demand dynamically within minutes, and usage is only monitored on a monthly basis.

> Your usage is available within the Altis Dashboard, and is updated throughout the day.
> ## How traffic is measured

Reviewer: Two blank lines before each heading:

```suggestion
Your usage is available within the Altis Dashboard, and is updated throughout the day.

## How traffic is measured
```

Altis measures your usage based on requests made to the WordPress-based content management system measured as **Page Views**.

> ## What is a page view?

Reviewer: ```suggestion

## What is a page view?
```

Page Views are any requests made to your WordPress-based content management system which invoke your application.

Views are intended to count users who access your site, and exclude static content (such as CSS, JS, and images) and known bots.

> ## How are views measured?

Reviewer: ```suggestion

## How are views measured?
```

> Altis uses traffic statistics generated by the Amazon CloudFront CDN to measure views. HTTP requests which invoke the WordPress-based backend are counted as a view.

Reviewer: ```suggestion
Altis uses traffic statistics generated by the [Amazon CloudFront CDN](./cdn/README.md) to measure views. HTTP requests which would invoke the WordPress-based backend are counted as a view.
```

> For customers who are using their own CDN, similar statistics generated by the load balancer are used to measure views instead.
> ## What traffic is excluded?

Reviewer: ```suggestion
For customers who are using their own CDN, similar statistics generated by the load balancer are used to measure views instead.

## What traffic is excluded?
```

Requests made by Altis automated systems and other known bots are excluded from views, even if they invoke the backend. Altis makes a best-effort attempt to exclude bot traffic, however cannot guarantee that all bots will be excluded. Altis also uses general firewall rules and best practices to block and mitigate malicious traffic.

> Enterprise environment tier customers may work with the Altis team to create custom firewall rules to block additional traffic.
> ## Why don’t my page views match my web analytics (such as Altis Analytics or Google Analytics)?

Reviewer: ```suggestion
Enterprise environment tier customers may work with the Altis team to create custom firewall rules to block additional traffic.

## Why don’t my page views match my web analytics (such as Altis Analytics or Google Analytics)?
```

Analytics services including Altis Analytics and Google Analytics measure traffic using JavaScript in users’ browsers. This allows these tools to collect detailed visitor data including location, browser versions, and page view duration. These traffic counts can differ from Page Views for a number of reasons, including users who block JavaScript or run ad blockers, browser caches, and custom filtering within web analytics tools.

> Your billing usage is always based on page views measured by the Altis backend and not measured by other tools.
> ## What data is collected for page view statistics?

Reviewer: ```suggestion
Your billing usage is always based on page views measured by the Altis backend and not measured by other tools.

## What data is collected for page view statistics?
```

> Page Views are only collected as an aggregate count and do not contain any personal information about your visitors.

Reviewer: ```suggestion
Page Views are collected as an aggregate count from the CDN access logs Altis stores. For further information, consult the Altis data processing addendum.
```

Reviewer 2: Does there need to be a link to the "data processing adendum"? OR is it an adendum to their contract?

Reviewer: It's part of the contract, yeah

> ## How much usage do I need?

Reviewer: ```suggestion

## How much usage do I need?
```

Altis will work with you to understand your unique needs, including looking at your historical usage and estimated traffic patterns, to right-size your environment for your requirements.

> ## What timeframe is used for page views?

Reviewer: ```suggestion

## What timeframe is used for page views?
```

Page views statistics are collected in near-realtime from the CloudFront CDN, and aggregated into monthly statistics. Your subscription includes usage either on a monthly or yearly basis.

> You can view page views for the current month-to-date and previous month within the Altis Dashboard.

Reviewer: ```suggestion
You can view estimated page views for the current month-to-date and previous months within the Altis Dashboard. The final page view numbers are available within the monthly reports.
```

> ## What happens if I use more than my included usage?

Reviewer: ```suggestion

## What happens if I use more than my included usage?
```

Included usage is based upon your typical traffic, ensuring you only pay for what you need.

If you’re on an annual billing plan, you may use up to 40% of your annual usage within a single month. This allows your usage to vary across the year with seasonal traffic patterns at no additional charge.

Additional overage is charged on a per-million basis above and beyond your included usage. For sustained levels of high traffic, contact your Account Manager to upgrade your plan.

If you sustain high overage across a long period, your Account Manager may reach out to discuss upgrading your plan.

> ## What if I receive traffic spikes?

Reviewer: ```suggestion

## What if I receive traffic spikes?
```

Altis environments are designed to automatically scale to meet traffic demands, and can generally handle large levels of traffic without problem. Automatic scaling measures your traffic across a five minute window, automatically provisioning new servers and increasing resources to ensure a good experience for your users.

Automatic scaling does not operate instantly and for large, instantaneous increases in traffic, your application may operate more slowly while the system scales to meet demand. This scaling period may take up to five minutes.

For applications with highly variable demand exceeding 10x or 100x typical traffic, such as blogs with high-traffic announcement posts, you can use the Traffic Burst feature of Altis. Traffic Burst prepares your site ahead of expected demand by pre-warming servers to reduce slowness during the scaling period, keeping a consistent user experience even as traffic increases. Contact your Account Manager for more information about Traffic Burst.

Usage during traffic spikes are counted as normal towards your usage, and do not incur any additional cost.

> ## What if my usage varies over a year?

Reviewer: ```suggestion

## What if my usage varies over a year?
```
